simpleflow/swf/executor.py

```diff
@@ -67,9 +67,10 @@ def run_fake_child_workflow_task(domain, task_list, workflow_type_name,
                                  workflow_type_version, workflow_id, input=None, result=None,
                                  child_policy=None, control=None, tag_list=None):
     conn = ConnectedSWFObject().connection
-    conn.start_child_workflow_execution(
-        workflow_type_name, workflow_type_version, workflow_id,
+    conn.start_workflow_execution(
```

Comment:
I should have double checked when we discussed that: it actually exists! http://boto.cloudhackers.com/en/latest/ref/swf.html#boto.swf.layer1_decisions.Layer1Decisions.start_child_workflow_execution
... but maybe not on the boto version we have for now?

To be honest I didn't test that method either when I introduce it, but I think we should stick with something that starts a child workflow, else we'll get an independent workflow execution and I don't get how SWF will connect the dots.

Maybe revert this commit and keep a big "TODO" for later, when we will work on child workflows?